edgegrid/log.go

```diff
@@ -12,41 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package edgegrid 
+package edgegrid
 
 import (
 	"bufio"
 	"fmt"
+	"net/http"
+	"net/http/httputil"
 	"os"
 	"strings"
-        "net/http"
-        "net/http/httputil"
+
 	log "github.com/sirupsen/logrus"
 )
 
 var logBuffer *bufio.Writer
 var LogFile *os.File
 var EdgegridLog *log.Logger
 
+// LogCorrelationID ID for header and footer of log file outputs
+var LogCorrelationID *string
```

Comment:
Shared variable will have trouble with concurrent calls.

Reply:
The same comments made in the provider AT-40 pull request apply here: I think the library should also be following a similar  [level][api][guid] <message> format. I would expect the common logging functions to be placed in the library itself and utilized in the provider.